tests/unit/test_lib_helpers.py

```diff
@@ -2,11 +2,42 @@
 from unittest import mock
 
 import pytest
-from netutils.lib_helpers import get_napalm_getters
+from netutils.lib_helpers import (
+    get_napalm_getters,
+    get_upgrade_path,
+    UPGRADE_PATHS,
+)
 
 
 @mock.patch("netutils.lib_helpers.HAS_NAPALM", False)
 def test_get_napalm_getters_napalm_not_installed():
     with pytest.raises(ImportError) as exc:
         get_napalm_getters()
     assert "Napalm must be install for this function to operate." == str(exc.value)
+
+
+def test_get_upgrade_path():
+    assert get_upgrade_path(
+        "9.1.6", "10.1.9", UPGRADE_PATHS["PANOS_FIRMWARE_LIST_OFFICIAL"]
+    ) == [  # pylint: disable=duplicate-code
```

Comment:
Probably shouldn't be running pylint on `test` files. Another option here is to change this list to one line (not sure if black formatted it like this) but that would get rid of the dup code error.

Reply:
Black formatted it like this, and just running the test suite from invoke hit this file with this error. If pylint shouldn't be run on test files, is there some way to exempt test_*.py files in the pylint config?